jetty-core/jetty-util/src/test/java/org/eclipse/jetty/util/resource/PathResourceTest.java

```diff
@@ -353,6 +356,25 @@ public void testNullCharEndingFilename(WorkDir workDir) throws Exception
         }
     }
 
+    @Test
+    public void testGetFileName(WorkDir workDir) throws IOException
+    {
+        Path tmpPath = workDir.getEmptyPathDir();
+        Path dir = tmpPath.resolve("foo-dir");
+        FS.ensureDirExists(dir);
+        Path file = dir.resolve("bar.txt");
+        Files.writeString(file, "This is bar.txt", StandardCharsets.UTF_8);
+
+        Resource baseResource = new PathResource(tmpPath);
+        assertThat(baseResource.getFileName(), endsWith(File.separator));
```

Comment:
I don't think you should use `File.separator` as the trailing character isn't platform-dependent.

Reply:
Using the same values as what `Path.getFileName()` return now.